### PR TITLE
Fix deprecation issues with new version of atom.

### PR DESCRIPTION
**Pull request by RodrigoEspinosa**
* Fix dependencies.
 * Replace jasmine.clock for window.setTimeout (since jasmine.clock is
   not installed by default). TODO: Should check if it is installed.
 * Remove jQuery function calls by vanilla function calls.

Resolves issue #21.
Resolves issue #20.
Resolves issue #19.
Resolves issue #18.

### CODE REVIEW

**Comment by rgbkrk** [comment]
OMG Awesome, thank you @RodrigoEspinosa.

File: lib/achievements.coffee [hunk 1]
```diff
@@ -9,6 +9,8 @@ module.exports =
       else
         new Achiever()
 
+    console.info "hellooooo"
```
Comment (rgbkrk) [inline lib/achievements.coffee]: I'm assuming this was for debugging and can be deleted now.

Comment (RodrigoEspinosa) [inline lib/achievements.coffee]: Oh my.. that's embarrassing, yes it can be deleted, sorry about that!

File: lib/achievements-view.coffee [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{View} = require 'atom'
+{WorkspaceView, View} = require 'atom-space-pen-views'
```
Comment (RodrigoEspinosa) [inline lib/achievements-view.coffee]: Also, this `WorkspaceView` is unnecessary.

File: lib/achievements.coffee (hunk 1 above)
Comment (rgbkrk) [inline lib/achievements.coffee]: All good. Want to delete it and add your name to the contributors list? After that I'll :ship: it.

Comment (RodrigoEspinosa) [inline lib/achievements.coffee]: Done, if find an issue in one of the specs, I can take a look at it in a few hours if you can't.

**Comment by rgbkrk** [comment]
Awesome, just tested it out. Looks great. Thank you so much @RodrigoEspinosa.

**Comment by rgbkrk** [comment]
```
14:07:48 (master=) kyle6475@puter ~/code/achievements$ apm publish patch
Preparing and tagging a new version ✓
Pushing v0.7.1 tag ✓
Publishing achievements@v0.7.1 ✓
```

**Comment by RodrigoEspinosa** [comment]
haha you're welcome! :smiley:

**Pull request merged**